### DISCUSSION TITLE
Can't get code block language dropdown to appear in Angular

### DISCUSSION THREAD

**Discussion by shadow1349** [Milkdown/milkdown#1246] [General]
Hey, I love the editor. I was just wondering how I could get the language dropdown to work for the code blocks in Angular. I copied the code from your website for the Angular example and I added prismjs to it along with the prism plugin. Another thing I noticed is that tailwindcss seems to continually override the styles unless I add the prose class from tailwind typography.

Here's a repo I made that simulates the issue.: https://github.com/shadow1349/milkdown-test

As an added note, I found this example on codesandbox of it working in Angular. I know it's an older version of milkdown but I'm wondering how to get it to work like this in newer versions.

https://codesandbox.io/p/sandbox/milkdown-angular-setup-wowuy?file=%2Fsrc%2Fapp%2Fapp.component.ts

Thanks!

**Comment by ousc** [comment]
Hello, would you like to try my project https://github.com/ousc/ng-milkdown?

this is the [code-block](https://github.com/ousc/ng-milkdown/blob/main/src/app/components/code-block.component.ts) component, and live demo: https://ousc.github.io/ng-milkdown/#/work-ground.

**Reply by shadow1349** [reply]
Thanks, I'll check it out!

**Reply by solos** [reply]
You could see the demo in the e2e dir. I use angular too.
demo: https://milkdown-storybook.vercel.app/?path=/story/components-code-block--javascript 
code: https://github.com/Milkdown/milkdown/blob/main/e2e/stories/components/code-block.stories.ts
